core.coffee

```diff
@@ -407,6 +407,19 @@
 			delete: (params) ->
 				return @request(params, method: 'DELETE')
 
+			batch: (data) ->
+				params =
+					method: 'POST'
+					url: '$batch'
+				overrides =
```

Comment:
I put them there because if I am not mistaken the `request` function strips away any unexpected property from the params object.